client/state/post-stats/actions.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
>Should this be under `state.stats.posts` or `state.posts.stats` instead? cc @aduth

Yes? 😆  

More helpfully, do we expect that we'll have more variations of stats? If so (and I suspect so), the first seems preferable as scoping stats data specific to the context of posts.